packagers/jam.go

```diff
@@ -47,15 +68,32 @@ func (j Jam) Execute(buildpackDir, output, version string, offline bool) error {
 	args := []string{
 		"pack",
 		command, filepath.Join(buildpackDir, buildpackOrExtensionToml),
-		"--output", output,
+		"--output", filepath.Join(jamOutput, fmt.Sprintf("%s.tgz", version)),
```

Comment:
can we encapsulate `fmt.Sprintf("%s.tgz", version)` into a variable - both for re-use but also to give it a useful name

Reply:
Sure I have no objections. Push something up soon.